sca/jumphosts/jumphost_main.tf

```diff
@@ -0,0 +1,112 @@
+and TransitGateway edits. Limits to VPCs that can be added are =to the limits of transit gateway.
+
+It is built to run in a region with three zones to use and will place services in 1a and 1c.  Modifications to other zones can be done.
+
+F5 Application Services will be deployed into the security VPC but if one wished they could also be deployed inside of the Application VPCs. 
+
+*/
+###############################################################################################################################################################################################################################################################
+
+resource "aws_security_group" "allow_incoming_jump_host" {
+  name        = "allow_incoming_jump_host"
+  description = "Allow Jumphost inbound traffic"
+  vpc_id      = var.vpcs.value.security
+
+  ingress {
+    # TLS (change to whatever ports you need)
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    # Please restrict your ingress to only necessary IPs and ports.
+    # Opening to 0.0.0.0/0 can lead to security vulnerabilities.
+    cidr_blocks = ["${var.my_public_ip}"]
+  }
+  ingress {
+    # RDP (change to whatever ports you need)
+    from_port   = 3389
+    to_port     = 3389
+    protocol    = "tcp"
+    # Please restrict your ingress to only necessary IPs and ports.
+    # Opening to 0.0.0.0/0 can lead to security vulnerabilities.
+    cidr_blocks = ["${var.my_public_ip}"]
+  }
+
+  egress {
+    from_port       = 0
+    to_port         = 0
+    protocol        = "-1"
+    cidr_blocks     = ["0.0.0.0/0"]
+  }
+}
+
+# Create a new instance of the latest Ubuntu 14.04 on an
+# t2.micro node with an AWS Tag naming it "HelloWorld"
+
+data "aws_ami" "ubuntu" {
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["ubuntu/images/hvm-ssd/ubuntu-*-18.04-amd64-server-*"]
+  }
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+
+  owners = ["099720109477"] # Canonical
+}
+
+resource "aws_instance" "Jumphost_AZ1" {
+  ami           = data.aws_ami.ubuntu.id
+  instance_type = "t2.micro"
+  key_name = var.jump_ssh_key
+  associate_public_ip_address = true
+  subnet_id = var.subnets.value.az1.security.mgmt
+  vpc_security_group_ids = ["${aws_security_group.allow_incoming_jump_host.id}"]
+  user_data = <<-EOF
+              #!/bin/bash
+              apt update
+              apt —yes —force-yes install xfce4 xfce4-goodies xorg dbus-x11 x11-xserver-utils
+              apt —yes —force-yes install xrdp
+              apt —yes —force-yes install awscli
+              apt —yes install python
+              snap install postman
+              EOF
+
+
+  tags = {
+    Name = "${var.project.value}_Jumphost"
+  }
+}
+
+resource "aws_instance" "Jumphost_AZ2" {
```

Comment:
why not use count = 2 and have one jumphost declaration?

Reply:
The jumphosts have different network attachments.

Reply:
I created a solution in https://github.com/f5devcentral/terraform-aws-f5-sca/pull/38. The PR pulls into this PR, not master.

Reply:
Parameterization added.